core/src/main/java/org/bitcoinj/core/TransactionInput.java

```diff
@@ -115,40 +115,55 @@ public static TransactionInput read(ByteBuffer payload, Transaction parentTransa
         TransactionOutPoint outpoint = TransactionOutPoint.read(payload);
         byte[] scriptBytes = Buffers.readLengthPrefixedBytes(payload);
         long sequence = ByteUtils.readUint32(payload);
-        return new TransactionInput(parentTransaction, scriptBytes, outpoint, sequence, null);
+        return new TransactionInput(parentTransaction, scriptBytes, outpoint, sequence);
     }
 
     public TransactionInput(@Nullable Transaction parentTransaction, byte[] scriptBytes,
                             TransactionOutPoint outpoint) {
-        this(parentTransaction, scriptBytes, outpoint, NO_SEQUENCE, null);
+        this(parentTransaction, null, scriptBytes, outpoint, NO_SEQUENCE, null, null);
+    }
+
+    public TransactionInput(@Nullable Transaction parentTransaction, byte[] scriptBytes,
+                            TransactionOutPoint outpoint, long sequence) {
+        this(parentTransaction, null, scriptBytes, outpoint, sequence, null, null);
     }
 
     public TransactionInput(@Nullable Transaction parentTransaction, byte[] scriptBytes,
                             TransactionOutPoint outpoint, @Nullable Coin value) {
-        this(parentTransaction, scriptBytes, outpoint, NO_SEQUENCE, value);
+        this(parentTransaction, null, scriptBytes, outpoint, NO_SEQUENCE, value, null);
```

Comment:
Can call the 5-arg constructor.

Reply:
Changed in https://github.com/bitcoinj/bitcoinj/compare/0b597887e953c7445e2ea0ab6c750143f30d882d..249b2a98453de7dfb90a36f10fc6b7dbf72af04d.